### PR TITLE
Decouple recipe persistence

### PR DESCRIPTION
**Pull request by Spaghet**
WIP

Related to #255, just takes the `environment_variable/desired`  and saves it to the couchdb environment_variable database.

It was a @TODO under the recipe_handler node to decouple so I figured I would just do that

### CODE REVIEW

File: launch/personal_food_computer_v2.launch [hunk 1]
```diff
@@ -29,6 +29,7 @@
       <param name="Kp" value="1" type="double"/>
     </node>
     <node pkg="openag_brain" type="recipe_handler.py" name="recipe_handler_1"/>
+    <node pkg="openag_brain" type="recipe_persistence" name="recipe_persistence_1"/>
```
Comment (rbaynes) [inline launch/personal_food_computer_v2.launch]: Should be `type="recipe_persistence.py"` add .py ?

**Comment by Spaghet** [comment]
Yeah, I'm trying to test it right now but it seems to run fine after we add the .py extension

**Comment by rwdavis513** [comment]
@Spaghet Could you add a test node to verify this is working correctly? I'd like to get in the habit of including tests for all new code that is submitted.

**Comment by Spaghet** [comment]
Do we have a specific testing tool we want to adhere to?
I see at least 3 we're using... nosetest, assert, unittest.

I'm going to get the new recipe interpreter running, and work with rbaynes on how ROStest works and get to testing this before I merge it in.

**Pull request merged**